backend/node_app/sql/v114.0_add_jaic_portfolio.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO public.portfolio(
+	name, description, user_ids, tags)
+	VALUES ('jaic', '', array[]::int[], array[]::text[]);
```

Comment:
The tags here should be the ones we use in the JAIC review so AI ENABLING AI ENABLED etc all those ones.